yoga/Yoga.c

```diff
@@ -229,8 +229,6 @@ static YGLogger gLogger = &YGDefaultLog;
 static inline const YGValue *YGComputedEdgeValue(const YGValue edges[YGEdgeCount],
                                                  const YGEdge edge,
                                                  const YGValue *const defaultValue) {
-  YG_ASSERT(edge <= YGEdgeEnd, "Cannot get computed value of multi-edge shorthands");
```

Comment:
`view.yoga.marginVertical = 20` is just syntax-sugar `[view.yoga setMarginVertical:20]`

Not sure how it translates for the Swift API, though.